sync/class.jetpack-sync-module-meta.php

```diff
@@ -2,11 +2,17 @@
 
 class Jetpack_Sync_Module_Meta extends Jetpack_Sync_Module {
 	private $meta_types = array( 'post', 'comment' );
+	private $post_meta_whitelist = array();
 
 	public function name() {
 		return 'meta';
 	}
 
+	public function set_defaults() {
+		$this->update_post_meta_whitelist();
```

Comment:
I think we can do `$this->post_meta_whitelist = Jetpack_Sync_Defaults::$default_whitelist_post_meta_keys;` directly here.